{
  "uid" : "8240ba6d3c493ea0",
  "name" : "确认密码为空",
  "fullName" : "test_regist.TestRegister#test_register",
  "historyId" : "ce6631e0b91488227d2b581bc9f6e8da",
  "time" : {
    "start" : 1646039634491,
    "stop" : 1646039634572,
    "duration" : 81
  },
  "status" : "failed",
  "statusMessage" : "AssertionError: assert {'code': 1003...': '确认密码不能为空'} == {'code': 1004...: '两次密码输入不一致'}\n  Differing items:\n  {'code': 1003} != {'code': 1004}\n  {'msg': '确认密码不能为空'} != {'msg': '两次密码输入不一致'}\n  Full diff:\n  - {'code': 1004, 'msg': '两次密码输入不一致'}\n  ?             ^          ^^  -- ^^\n  + {'code': 1003, 'msg': '确认密码不能为空'}...\n  \n  ...Full output truncated (2 lines hidden), use '-vv' to show",
  "statusTrace" : "self = <autotest03.testcases.test_my_interface01.test_regist.TestRegister object at 0x053D1040>\ncase = <comms.excel_utils.CaseData object at 0x04AB0CD0>\n\n    @allure.severity(\"critical\")\n    @pytest.mark.parametrize(\"case\", cases)\n    def test_register(self, case):\n        allure.dynamic.title(case.case_title)\n    \n        db = DBUtils()\n        # 正确流程\n        if case.case_id == 1:\n            username = eval(case.case_data)['username']  # 获取传入的用户名\n            db.cud('delete from tb_user where name = %s', (username,))\n            db.close()\n    \n        response = requests.post(url=case.url, data=eval(case.case_data))\n        res_body = response.json()\n    \n        try:\n            assert eval(case.expect) == res_body\n        except AssertionError as e:\n            ReadExcel.write_data(DATA_FILE, 'Register', case.case_id, 7, '失败')\n            logger.error('测试编号{},测试用例标题:{},执行失败,实际结果为:{}!'.format(case.case_id, case.case_title, res_body))\n            logger.exception(e)\n>           raise e\n\ntest_regist.py:45: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <autotest03.testcases.test_my_interface01.test_regist.TestRegister object at 0x053D1040>\ncase = <comms.excel_utils.CaseData object at 0x04AB0CD0>\n\n    @allure.severity(\"critical\")\n    @pytest.mark.parametrize(\"case\", cases)\n    def test_register(self, case):\n        allure.dynamic.title(case.case_title)\n    \n        db = DBUtils()\n        # 正确流程\n        if case.case_id == 1:\n            username = eval(case.case_data)['username']  # 获取传入的用户名\n            db.cud('delete from tb_user where name = %s', (username,))\n            db.close()\n    \n        response = requests.post(url=case.url, data=eval(case.case_data))\n        res_body = response.json()\n    \n        try:\n>           assert eval(case.expect) == res_body\nE           AssertionError: assert {'code': 1003...': '确认密码不能为空'} == {'code': 1004...: '两次密码输入不一致'}\nE             Differing items:\nE             {'code': 1003} != {'code': 1004}\nE             {'msg': '确认密码不能为空'} != {'msg': '两次密码输入不一致'}\nE             Full diff:\nE             - {'code': 1004, 'msg': '两次密码输入不一致'}\nE             ?             ^          ^^  -- ^^\nE             + {'code': 1003, 'msg': '确认密码不能为空'}...\nE             \nE             ...Full output truncated (2 lines hidden), use '-vv' to show\n\ntest_regist.py:40: AssertionError",
  "flaky" : false,
  "newFailed" : false,
  "newBroken" : false,
  "newPassed" : false,
  "retriesCount" : 0,
  "retriesStatusChange" : false,
  "beforeStages" : [ ],
  "testStage" : {
    "status" : "failed",
    "statusMessage" : "AssertionError: assert {'code': 1003...': '确认密码不能为空'} == {'code': 1004...: '两次密码输入不一致'}\n  Differing items:\n  {'code': 1003} != {'code': 1004}\n  {'msg': '确认密码不能为空'} != {'msg': '两次密码输入不一致'}\n  Full diff:\n  - {'code': 1004, 'msg': '两次密码输入不一致'}\n  ?             ^          ^^  -- ^^\n  + {'code': 1003, 'msg': '确认密码不能为空'}...\n  \n  ...Full output truncated (2 lines hidden), use '-vv' to show",
    "statusTrace" : "self = <autotest03.testcases.test_my_interface01.test_regist.TestRegister object at 0x053D1040>\ncase = <comms.excel_utils.CaseData object at 0x04AB0CD0>\n\n    @allure.severity(\"critical\")\n    @pytest.mark.parametrize(\"case\", cases)\n    def test_register(self, case):\n        allure.dynamic.title(case.case_title)\n    \n        db = DBUtils()\n        # 正确流程\n        if case.case_id == 1:\n            username = eval(case.case_data)['username']  # 获取传入的用户名\n            db.cud('delete from tb_user where name = %s', (username,))\n            db.close()\n    \n        response = requests.post(url=case.url, data=eval(case.case_data))\n        res_body = response.json()\n    \n        try:\n            assert eval(case.expect) == res_body\n        except AssertionError as e:\n            ReadExcel.write_data(DATA_FILE, 'Register', case.case_id, 7, '失败')\n            logger.error('测试编号{},测试用例标题:{},执行失败,实际结果为:{}!'.format(case.case_id, case.case_title, res_body))\n            logger.exception(e)\n>           raise e\n\ntest_regist.py:45: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <autotest03.testcases.test_my_interface01.test_regist.TestRegister object at 0x053D1040>\ncase = <comms.excel_utils.CaseData object at 0x04AB0CD0>\n\n    @allure.severity(\"critical\")\n    @pytest.mark.parametrize(\"case\", cases)\n    def test_register(self, case):\n        allure.dynamic.title(case.case_title)\n    \n        db = DBUtils()\n        # 正确流程\n        if case.case_id == 1:\n            username = eval(case.case_data)['username']  # 获取传入的用户名\n            db.cud('delete from tb_user where name = %s', (username,))\n            db.close()\n    \n        response = requests.post(url=case.url, data=eval(case.case_data))\n        res_body = response.json()\n    \n        try:\n>           assert eval(case.expect) == res_body\nE           AssertionError: assert {'code': 1003...': '确认密码不能为空'} == {'code': 1004...: '两次密码输入不一致'}\nE             Differing items:\nE             {'code': 1003} != {'code': 1004}\nE             {'msg': '确认密码不能为空'} != {'msg': '两次密码输入不一致'}\nE             Full diff:\nE             - {'code': 1004, 'msg': '两次密码输入不一致'}\nE             ?             ^          ^^  -- ^^\nE             + {'code': 1003, 'msg': '确认密码不能为空'}...\nE             \nE             ...Full output truncated (2 lines hidden), use '-vv' to show\n\ntest_regist.py:40: AssertionError",
    "steps" : [ ],
    "attachments" : [ {
      "uid" : "2c01e435249a552a",
      "name" : "log",
      "source" : "2c01e435249a552a.txt",
      "type" : "text/plain",
      "size" : 1367
    } ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "hasContent" : true,
    "attachmentsCount" : 1,
    "shouldDisplayMessage" : true
  },
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "feature",
    "value" : "注册接口测试"
  }, {
    "name" : "severity",
    "value" : "critical"
  }, {
    "name" : "suite",
    "value" : "test_regist"
  }, {
    "name" : "subSuite",
    "value" : "TestRegister"
  }, {
    "name" : "host",
    "value" : "DESKTOP-V37VTV5"
  }, {
    "name" : "thread",
    "value" : "65612-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "test_regist"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ {
    "name" : "case",
    "value" : "<comms.excel_utils.CaseData object at 0x04AB0CD0>"
  } ],
  "links" : [ ],
  "hidden" : true,
  "retry" : true,
  "extra" : {
    "categories" : [ ],
    "tags" : [ ]
  },
  "source" : "8240ba6d3c493ea0.json",
  "parameterValues" : [ "<comms.excel_utils.CaseData object at 0x04AB0CD0>" ]
}